Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Description: on Debian, we don't need vendored versions of dependencies.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -89,7 +89,3 @@
     "dep:sha2",
     "dep:hkdf",
 ]
-vendored = [
-    "dbus/vendored",
-    "openssl?/vendored",
-]
